app/build.gradle

```diff
@@ -24,9 +24,10 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'com.google.android.gms:play-services:11.4.2'
+    //noinspection UseOfBundledGooglePlayServices
+    implementation 'com.google.android.gms:play-services:11.6.2'
```

Comment:
Yikes!  Rather than ignoring the inspection how about we just fix this one by removing this line and uncommenting the one below.

Reply:
Ok, changed